doc/DeveloperGuide.md

```diff
@@ -88,6 +88,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename an existing tag
+
+**Preconditions**
+
+1. AddressBook is populated with at least one person
+2. There exists at least one person with a tag
+
+**Guarantees**
+
+1. All instances of the specified tag will be renamed
+
+**MSS**
+
+1. User specifies a tag to be renamed
+2. AddressBook shows a list of persons with the specified tag
+3. User inputs a new tag name
+4. AddressBook requests confirmation of the change
+5. User confirms the change
+4. AddressBook renames all instances of the specified tag <br>
```

Comment:
The numbering should be `6.`

On a side note, markdown allows us something like this:

```
1.
1.
1.
1.
```
 is rendered as 
```
1.
2.
3.
4.
```

This helps user to avoid renumbering when some point in middle is removed later on